pkg/util/filterutil/filterutil.go

```diff
@@ -65,8 +68,36 @@ const (
 	ParamFilterAnnotations = "filterAnnotations"
 	ParamFilterLabels      = "filterLabels"
 	ParamFilterServices    = "filterServices"
+
+	ParamFilterAccounts      = "filterAccounts"
+	ParamFilterCategories    = "filterCategories"
+	ParamFilterNames         = "filterNames"
+	ParamFilterProjects      = "filterProjects"
+	ParamFilterProviders     = "filterProviders"
+	ParamFilterProviderIDs   = "filterProviderIDs"
+	ParamFilterProviderIDsV2 = "filterProviderIds"
```

Comment:
Can we not have a common filterProviderIDs for both v1 and v2? Is there a reason why this is different?

Reply:
This is a good question, thank you. I pulled this code from the parallel KC cloud implementation (https://github.com/kubecost/kubecost-cloud/blob/194b58017aa632cfeb0ee6bd61804ad7a17d5349/internal/util/filterutil/filterutil.go#L61).

KCM doesn't appear to have had this "v2" param (https://github.com/kubecost/kubecost-cost-model/pull/1581/files#diff-a6a736ce66bb7870d4233a16dbdecdd398a25042f790a5663af4d67b6f5e4d41L46-L61).

@mbolt35 what's up with the v2 provider ID filter? should it be included in Opencost?